wpimath/src/main/java/edu/wpi/first/math/kinematics/ChassisSpeeds.java

```diff
@@ -362,6 +362,17 @@ public ChassisSpeeds div(double scalar) {
         vxMetersPerSecond / scalar, vyMetersPerSecond / scalar, omegaRadiansPerSecond / scalar);
   }
 
+  /**
+   * Checks if the ChassisSpeeds is equivalent to the other provided ChassisSpeeds.
+   *
+   * @param other The ChassiSpeeds object to compare to.
+   */
+  public Boolean equals(ChassisSpeeds other) {
```

Comment:
Why does this return a boxed Boolean? Shouldn't this override the `equals(Object)` method?

Reply:
I did not know that existed...
Would this be a better way of doing it?